docs/console-application-issues.md

```diff
@@ -17,42 +17,47 @@ ms.assetid: a561fbdd-b50d-4687-92d7-735377a7991d
-| [**CopyFile**](https://msdn.microsoft.com/library/windows/desktop/aa363851)                       | [**GetFileAttributes**](https://msdn.microsoft.com/library/windows/desktop/aa364944)       | [**LoadLibrary**](https://msdn.microsoft.com/library/windows/desktop/ms684175)                 |
-| [**CreateDirectory**](https://msdn.microsoft.com/library/windows/desktop/aa363855)         | [**GetFullPathName**](https://msdn.microsoft.com/library/windows/desktop/aa364963)           | [**LoadLibraryEx**](https://msdn.microsoft.com/library/windows/desktop/ms684179)             |
-| [**CreateFile**](https://msdn.microsoft.com/library/windows/desktop/aa363858)                   | [**GetModuleFileName**](https://msdn.microsoft.com/library/windows/desktop/ms683197)       | [**MoveFile**](https://msdn.microsoft.com/library/windows/desktop/aa365239)                       |
-| [**CreateProcess**](https://msdn.microsoft.com/library/windows/desktop/ms682425)             | [**GetModuleHandle**](https://msdn.microsoft.com/library/windows/desktop/ms683199)           | [**MoveFileEx**](https://msdn.microsoft.com/library/windows/desktop/aa365240)                   |
-| [**DeleteFile**](https://msdn.microsoft.com/library/windows/desktop/aa363915)                   | [**GetSystemDirectory**](https://msdn.microsoft.com/library/windows/desktop/ms724373)     | [**OpenFile**](https://msdn.microsoft.com/library/windows/desktop/aa365430)                       |
-| [**FindFirstFile**](https://msdn.microsoft.com/library/windows/desktop/aa364418)             | [**GetTempFileName**](https://msdn.microsoft.com/library/windows/desktop/aa364991)           | [**RemoveDirectory**](https://msdn.microsoft.com/library/windows/desktop/aa365488)         |
-| [**FindNextFile**](https://msdn.microsoft.com/library/windows/desktop/aa364428)               | [**GetTempPath**](https://msdn.microsoft.com/library/windows/desktop/aa364992)                   | [**SearchPath**](https://msdn.microsoft.com/library/windows/desktop/aa365527)                   |
-| [**GetCurrentDirectory**](https://msdn.microsoft.com/library/windows/desktop/aa364934) | [**GetVolumeInformation**](https://msdn.microsoft.com/library/windows/desktop/aa364993) | [**SetCurrentDirectory**](https://msdn.microsoft.com/library/windows/desktop/aa365530) |
-| [**GetDiskFreeSpace**](https://msdn.microsoft.com/library/windows/desktop/aa364935)       | [**GetWindowsDirectory**](https://msdn.microsoft.com/library/windows/desktop/ms724454)   | [**SetFileAttributes**](https://msdn.microsoft.com/library/windows/desktop/aa365535)     |
-| [**GetDriveType**](https://msdn.microsoft.com/library/windows/desktop/aa364939)               |                                                       |                                                     |
-||
-||
-||
-||
-||
-||
-||
-
-
+:::row:::
+    :::column:::
```

Comment:
I don't recognize this notation and GitHub doesn't seem to understand it for markdown formatting. Is this something special in our doc build system?

Reply:
I'm pretty sure it is, IIRC there's some similar stuff in the TerminalDocs repo, but @cinnamon-msft can verify

Reply:
Multi-column coding is explained here:

https://review.docs.microsoft.com/en-us/help/contribute/validation-ref/column-header-missing?branch=master

Reply:
Markdown tables with missing headers is probably going to be elevated to a warning next month, and my group has been working recently to fix those in various repos as part of another project. Since I saw an instance in this file, I just decided to correct that.

Reply:
Aha, OK. I just wasn't familiar with that notation nor the fact that our docs build extensions are apparently... documented publicly on that site. Wonderful. Thanks for clearing it up for me.